docs/chaosiq-on-docker/run-with-chaosiq.md

```diff
@@ -0,0 +1,67 @@
+
+## Running with ChaosIQ
+
+To connect the image to ChaosIQ when it runs the experiment we will have to provide a settings file. To do this you will need a local environment setup to run the [Chaos Toolkit][ChaosToolkit] with the [ChaosIQ extension][ChaosIQCloud]. For full details of setting this up see [ChaosIQ prerequisites][InstallCtk]. You will also need a Token from ChaosIQ, if you don't already have a token that you can use login to [ChaosIQ][ChaosIQ] and visit the [Tokens page][ChaosIQTokens] to generate a new Token (remember to capture the Token in your clipboard).
+
+To create a settings file for ChaosIQ run the following command:
```

Comment:
why are we explaining this here.? Isn't it already documented elsewhere?

Reply:
updated and added a cross reference